css/scrollbar-gutter/demos/bad/index.html

```diff
@@ -0,0 +1,198 @@
+        на Земле в доисторический период. Те были поистине гигантскими - весом в
+        несколько тонн. Они населяли нашу планету в последнюю ледниковую эпоху и
+        исчезли примерно 11 тыс. лет назад.
+      </p>
+      <p>
+        Ходили они по земле, а чтобы достать листья, которыми они питались,
+        становились на задние лапы.
+      </p>
+      <p>
+        "Изменения, которые произошли с ленивцами, - это результат того, что они
+        начали жить на деревьях и в питании перешли почти исключительно на
+        листья", - говорит Камила Маццони.
+      </p>
+      <p>
+        "Лиственная диета весьма скудна на питательные вещества, калорий в
+        листве очень мало. Из-за этого у ленивцев очень медленно идущий обмен
+        веществ".
+      </p>
+      <p>
+        Частично это объясняется тем, где они живут. Все шесть видов ленивцев
+        обитают в тропических лесах. Там жарко и влажно, так что нужда в
+        движении, согревающем мышцы и сердечно-сосудистую систему, отпадает. Вот
+        ленивцы и не напрягаются.
+      </p>
+      <p>
+        В том, чтобы быть млекопитающим, есть свои преимущества. Способность
+        регулировать внутреннюю температуру тела позволяет жить в куда более
+        холодном климате, чем могут рептилии или другие холоднокровные животные.
+      </p>
+      <p>
+        Но теплокровная природа имеет и некоторые неудобства. Чтобы поддерживать
+        в рабочем состоянии эту жадную на энергию систему, млекопитающим
+        приходится много есть и поддерживать мышцы в разогретом состоянии, чтобы
+        без проблем передвигаться.
+      </p>
+      <p>
+        Эта эндотермическая адаптация позволяет млекопитающим существовать почти
+        в любых условиях обитания на Земле, в том числе на обоих полюсах.
+      </p>
+      <p>
+        Но кое-где некоторые млекопитающие отказались от преимуществ, подаренных
+        им природой. Они выбрали энергосберегающий стиль жизни, больше
+        напоминающий образ жизни холоднокровных животных.
+      </p>
+      <p>
+        "Система терморегуляции, которой обладает большинство млекопитающих,
+        требует много энергии, - говорит Маццони. - Поскольку у ленивцев такой
+        системы нет, им энергии нужно куда меньше. Однако это означает, что они
+        могут жить только в тропиках, причём не высоко в горах, где температура
+        может падать. Впрочем, двупалые ленивцы иногда забираются повыше в горы
+        в Коста-Рике".
+      </p>
+    </div>
+    <script>
+      const dialog = document.querySelector(".dialog");
+      const openButton = document.querySelector(".open-button");
+      const closeButton = document.querySelector(".close-button");
+
+      openButton.addEventListener("click", () => {
+        dialog.showModal();
+        document.body.style.overflowY = "hidden";
+      });
+
+      closeButton.addEventListener("click", () => {
+        dialog.close();
+        document.body.style.overflowY = "auto";
+      });
```

Comment:
```suggestion
      const dialog = document.querySelector('.dialog')
      const openButton = document.querySelector('.open-button')
      const closeButton = document.querySelector('.close-button')

      openButton.addEventListener('click', () => {
        dialog.showModal()
        document.body.style.overflowY = 'hidden'
      })

      closeButton.addEventListener('click', () => {
        dialog.close()
        document.body.style.overflowY = 'auto'
      })
```

Подогнала под наш кодстайл